code/modules/antagonists/clockcult/clock_items/clock_weapons/ratvarian_spear.dm

```diff
@@ -56,8 +57,13 @@
 				L.visible_message("<span class='warning'>[src] bounces off of [L], as if repelled by an unseen force!</span>")
 		else if(!..())
 			if(!L.anti_magic_check())
-				if(issilicon(L) || iscultist(L))
```

Comment:
removing the knockdown entirely is a bit silly maybe make it just knockdown(1) or something so it knocks down without doing much stamina